scripts/shell.nix

```diff
@@ -0,0 +1,8 @@
+with (import <nixpkgs> { });
```

Comment:
Not sure, but I think this doesn't actually use the sources.json. @tiagolobocastro might be able to tell.

Reply:
Yes it does, check in the workflow where we do prepopulate.

Reply:
this is not needed for the CI as per the changes I see, though it would be needed if we want to use it locally, otherwise we'd all get our own versions

Reply:
Have changed it